visualvm/libs.profiler/lib.profiler/src/org/graalvm/visualvm/lib/jfluid/classfile/SameNameClassGroup.java

```diff
@@ -57,12 +57,12 @@
 public class SameNameClassGroup {
     //~ Instance fields ----------------------------------------------------------------------------------------------------------
 
-    private List classes;
+    private List<BaseClassInfo> classes;
 
     //~ Constructors -------------------------------------------------------------------------------------------------------------
 
     public SameNameClassGroup() {
-        classes = new ArrayList(4); // Hope we are not going to have too many class versions...
+        classes = new ArrayList<>(4); // Hope we are not going to have too many class versions...
```

Comment:
Initialization could be moved to declaration, then the constructor could even be removed.